physical/postgresql.go

```diff
@@ -74,6 +156,11 @@ func newPostgreSQLBackend(conf map[string]string, logger log.Logger) (Backend, e
 			"UNION SELECT DISTINCT substring(substr(path, length($1)+1) from '^.*?/') FROM " +
 			quoted_table + " WHERE parent_path LIKE concat($1, '%')",
 		logger: logger,
+
```

Comment:
pinging @knz which is one of the master & commanders of CockroachDB in case he says this is not supported yet and if he has an idea of how could be rewritten in a way which it is.

Reply:
I'm not exactly sure what the question is? You already use `pq.QuoteIdentifier` properly; you could format your query with `fmt.Sprintf("SELECT value FROM %s WHERE path = $1 AND key = $2", quoted_table)` and that should work. Was there any reason to not do that?

Meanwhile regarding the `list_query`:

- the UNION should work; however
- I'm not sure what you are doing with `substring( ... from '^.*?/')`. This does't look right; `substring` does not work with regular expressions. There are `regexp_xxx()` functions that can help though.

Reply:
It was for the `UNION SELECT DISTINCT` thingy so thanks for the quick response!